src/internal/util.csx

```diff
@@ -30,10 +30,11 @@ $@"{Cyan(color)}Usage: {Default(color)}{BrightYellow(color)}<script-runner> {Def
 {Cyan(color)}script-file: {Default(color)}Path to a script. E.g. build.csx.
 
 {Cyan(color)}options:{Default(color)}
- {White(color)}-D          {Default(color)}Display the targets and dependencies, then exit
- {White(color)}-T          {Default(color)}Display the targets, then exit
- {White(color)}-n          {Default(color)}Do a dry run without executing actions
- {White(color)}--no-color  {Default(color)}Disable colored output
+ {White(color)}-D                  {Default(color)}Display the targets and dependencies, then exit
+ {White(color)}-T                  {Default(color)}Display the targets, then exit
+ {White(color)}-n                  {Default(color)}Do a dry run without executing actions
+ {White(color)}--no-color          {Default(color)}Disable colored output
+ {White(color)}--skip-dependencies {Default(color)}Do not run targets' dependencies

```

Comment:
To make this usable from the CLI, I think it needs to be `-s`. We can look into having both `-` and `--` aliases for a specific option later on.

Reply:
I do not follow "usable from the CLI". I guess `--no-color` isn't?

Reply:
I.e. I would hate to have to type out `--skip-dependencies` in the CLI.

`--no-color` isn't something that people would typically use from the CLI. It's more something that you would configure in a CI server, to prevent the coloured output getting mangled in the build server UI. IIRC older versions of TeamCity did this, and that's why I added the option.

Reply:
> I.e. I would hate to have to type out `--skip-dependencies` in the CLI.